src/app/commands/bounty/IOU.ts

```diff
@@ -0,0 +1,210 @@
+import {
+    CommandContext,
+    CommandOptionType,
+    SlashCommand,
+    SlashCreator,
+} from 'slash-create';
+import { handler } from '../../activity/bounty/Handler';
+
+import ValidationError from '../../errors/ValidationError';
+import Log, { LogUtils } from '../../utils/Log';
+import { Request } from '../../requests/Request';
+import { Activities } from '../../constants/activities';
+import { CreateRequest } from '../../requests/CreateRequest';
+import { PaidRequest } from '../../requests/PaidRequest';
+import { DeleteRequest } from '../../requests/DeleteRequest';
+import { ListRequest } from '../../requests/ListRequest';
+import { HelpRequest } from '../../requests/HelpRequest';
+import AuthorizationError from '../../errors/AuthorizationError';
+import DiscordUtils from '../../utils/DiscordUtils';
+import { guildIds } from '../../constants/customerIds';
+
+
+export default class IOU extends SlashCommand {
+    constructor(creator: SlashCreator) {
+        super(creator, {
+            name: 'iou',
+            description: 'List, Create, Delete, and mark IOUs paid',
+            //TODO: make this dynamic? - can pull guildId list by querying mongo from app.ts on startup
+            guildIDs: guildIds,
+            options: [
+                {
+                    name: Activities.create,
+                    type: CommandOptionType.SUB_COMMAND,
+                    description: 'Create a new IOU',
+                    options: [
+                        {
+                            name: 'reward',
+                            type: CommandOptionType.STRING,
+                            description: 'What is the reward? (i.e 100 BANK)',
+                            required: true,
+                        },
+                        {
+                            name: 'owed-to',
```

Comment:
I think owed-to should come first - was there a reason you were thinking of reward first?

when I say it out loud  "I owe chkaloon 100 BANK for being awesome", which is where my preference comes from so <user> <reward> <message>

Reply:
Done